safekeeper/src/timelines_global_map.rs

```diff
@@ -327,16 +327,21 @@ impl GlobalTimelines {
     }
 
     /// Cancels timeline, then deletes the corresponding data directory.
-    pub async fn delete_force(ttid: &TenantTimelineId) -> Result<TimelineDeleteForceResult> {
+    /// If only_local, doesn't remove WAL segments in remote storage.
+    pub async fn delete(
+        ttid: &TenantTimelineId,
+        only_local: bool,
+        conf: &SafeKeeperConf,
```

Comment:
This is redundant, you can get conf with `self.get_global_config()`

Reply:
Yeah, forgot about it, removed.